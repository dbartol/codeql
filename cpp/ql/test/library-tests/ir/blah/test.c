typedef struct _GSList GSList;

struct _GSList
{
  void* data;
  GSList *next;
};

void foo(void) {
    GSList* p;
}