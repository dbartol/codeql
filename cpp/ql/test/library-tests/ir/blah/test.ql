import cpp

from Type type
where type.toString() = "_GSList"
select type