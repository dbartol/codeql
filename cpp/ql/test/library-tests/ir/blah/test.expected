| test.c:3:8:3:14 | _GSList |
